packages/account/src/Constants/onfido.js

```diff
@@ -341,4 +342,7 @@ const getOnfidoPhrases = () => ({
     },
 });
 
-export default getOnfidoPhrases;
+export const LANGUAGE_CODES = {
+    ID: 'id_ID',
+    EN: 'en_US',
+};
```

Comment:
What is the usecase of these language codes? Do we only need these 2?

Reply:
En is just default value,
as for Indonesia - we have to make sure to pass Indonesian as 'id_ID', because onfido doesn’t support two letters code for indonesia